.github/workflows/tests.yml

```diff
@@ -8,6 +8,10 @@ on:
     branches:
     - master
 
+env:
+  MAMBA_ROOT_PREFIX: ~/micromamba
+  MAMBA_EXE: $(pwd)/micromamba
```

Comment:
In the `jobs` section it only works for macOS for some reason. At the top most level Linux can find the env var but micromamba crashes. (@wolfv I'm pinging you just for awareness as  Idebug this, please let me know if I should stop!)